tests/integration/test_question_pages.py

```diff
@@ -11,8 +13,26 @@ def test_get_question_summary_page(django_app):
     question = section.question_set.first()
     answer = question.answer_set.first()
     question_summary_url = f"/consultations/{consultation.slug}/sections/{section.slug}/questions/{question.slug}"
+    question_page = django_app.get(question_summary_url)
+    page_content = html.unescape(str(question_page.content))
+
+    assert question.text in page_content
+    assert answer.theme.summary in page_content
 
     question_page = django_app.get(question_summary_url)
+    assert question.text in page_content
+    assert answer.theme.summary in page_content
```

Comment:
This is duplicated from above

Reply:
Ah, yes, missed that when resolving conflicts!